src/TogglePassword/src/Form/TogglePasswordTypeExtension.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TogglePassword\Form;
+
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Félix Eymonot <felix.eymonot@alximy.io>
+ */
+final class TogglePasswordTypeExtension extends AbstractTypeExtension
+{
+    public static function getExtendedTypes(): iterable
+    {
+        return [PasswordType::class];
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefined([
+            'toggle',
+            'hidden_label',
+            'visible_label',
+            'hidden_icon',
+            'visible_icon',
+        ]);
```

Comment:
As you use `setDefaults()`, I don't think using `setDefined()` is useful.

Reply:
If i remove `setDefined()` and keep only `setDefaults()`, the options are not found by the `OptionsResolver` 😅

Reply:
What if you move the `setDefaults()` call above the `setAllowedTypes()` calls?

Reply:
Yes indeed, make sense and work in that order 👍🏼